packages/forma-36-react-components/package.json

```diff
@@ -31,6 +31,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.3.4",
+    "@contentful/f36-polymorphic": "^4.0.0",
+    "@contentful/f36-primitive": "^4.0.0",
```

Comment:
Can't we have both components in one package? 

I'd rather introduce `f36-core` and collocate it with several other components.